### ISSUE TITLE
Can't reproduce your accuracy for Urbansound8K dataset

### ISSUE THREAD

**Issue by ltshan** [karoldvl/paper-2015-esc-dataset#1]
With your CNN architecture(2 Relu CNN + 2 connected layer + 1 output layer), My experiment for Urbansound8K fails(training accuracy kept as about 0.11 41, seems gradients become 0). 

Audio features already extracted as log melspec and its delta, whose shape is (-1, 60, 41, 2) before input network.  

can you help check it or do you have the code of keras/tensorflow now?  thank you
